src/components/common/YourCompany/BusinessStartDate.vue

```diff
@@ -1,5 +1,16 @@
 <template>
   <section id="business-start-date">
+    <!-- <v-row no-gutters class="mb-6">
+      <v-col cols="3" class="pr-2">
+        <label :class="{'error-text': invalidSection}">
+          <strong>Registration Date</strong>
+        </label>
+      </v-col>
+      <v-col cols="9">
+        <span class="info-text mr-1">{{ businessFoundingDate }}</span>
+      </v-col>
+    </v-row> -->
```

Comment:
For testing (and I'm expecting some changes from Yui).

I'll try to remember to delete this before merging this PR 😁